crates/util/python-utils/src/constants.rs

```diff
@@ -0,0 +1,70 @@
+use std::sync::LazyLock;
+
+use bdk_wallet::bitcoin::{
+    key::Parity,
+    secp256k1::{PublicKey, SecretKey, SECP256K1},
+    Amount, Network, XOnlyPublicKey,
+};
+use shrex::hex;
+
+/// Magic bytes to add to the metadata output in transactions to help identify them.
+pub const MAGIC_BYTES: &[u8; 11] = b"alpenstrata";
+
+/// Number of blocks after bridge in transaction confirmation that the recovery path can be spent.
+pub(crate) const RECOVER_DELAY: u32 = 1008;
+
+/// 10 BTC + 0.01 to cover fees in the following transaction where the operator spends it into the
+/// federation.
+pub(crate) const BRIDGE_IN_AMOUNT: Amount = Amount::from_sat(1_001_000_000);
+
+/// Bridge outs are enforced to be exactly 10 BTC
+#[allow(dead_code)] // TODO: Remove this when bridge out is implemented
+pub(crate) const BRIDGE_OUT_AMOUNT: Amount = Amount::from_int_btc(10);
```

Comment:
Need to create an issue to make these not hardcoded.  I know this is blocked on making them not hardcoded in the bridge code and derived from params. :frowning:

Reply:
https://alpenlabs.atlassian.net/browse/STR-531